Competition2017/src/competition/subsystems/shift/commands/ShiftGearCommand.java

```diff
@@ -0,0 +1,43 @@
+package competition.subsystems.shift.commands;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.shift.ShiftSubsystem;
+import competition.subsystems.shift.ShiftSubsystem.Gear;
+import xbot.common.command.BaseCommand;
+
+public class ShiftGearCommand extends BaseCommand {
```

Comment:
In the past it's been a cheap way to quickly see which gear we're in by which command is currently running on that subsystem. Also allows us to have a default command to start off in (and you don't want a default command that finishes or it will just churn endlessly)